satsense/image.py

```diff
@@ -74,7 +93,7 @@ def shallow_copy_range(self, x_range, y_range, pad=True):
             self._normalized_image = normalize_image(
                 self.raw, self.bands, **self._normalization_parameters)
 
-        im._normalized_image = self._normalized_image[x_range, y_range]
+            im._normalized_image = self._normalized_image[x_range, y_range]
```

Comment:
Why is this changed? Was it a bug?

Reply:
Yes, I think I have had errors on this line. Changed this some time ago.